config/test.exs

```diff
@@ -1,12 +1,10 @@
 use Mix.Config
 
-config :fusion_auth, :tesla, adapter: Tesla.Mock
-
 config :logger, level: :error
 
 config :fusion_auth,
+  api_url: System.get_env("FUSION_AUTH_TEST_URL"),
```

Comment:
Can't we use the same env variable here of `FUSION_AUTH_URL`?

Reply:
the `FUSION_AUTH_URL` is what the app uses to configure the port, but the test url is what the tests use to point at the service running in the container, so `FUSION_AUTH_URL="http://localhost:9011"` and `FUSION_AUTH_TEST_URL="http://localhost:29012"`

Reply:
Gotcha. Just thought we could leverage the same variable name for regular dev vs. testing as we are for other services such as Postgres and Elastic. If we need the TEST variable then so be it